### DISCUSSION TITLE
Plain Vike returns just an index.html (no vike present)

### DISCUSSION THREAD

**Discussion by knownasilya** [vikejs/bati#160] [Q&A] [answered]
Is this expected?

**Comment by magne4000** [comment]
Yes it was, but adding a little bit more couldn't hurt I guess.
A commented [onRenderHtml](https://vike.dev/onRenderHtml) and [onRenderClient](https://vike.dev/onRenderClient) could be a good introduction.

**Reply by knownasilya** [reply]
I used https://batijs.github.io/ and clicked "Plain Vike" does that not give you the right flags?

**Reply by magne4000** [reply]
Yes it does, but "Plain Vike" is just an empty shell really. You still need a renderer to display anything, and without a `vike-*` package to do that, you need to do it yourself. Does that make sense?

**Reply by magne4000** [reply]
I guess it could also be in [HTML-only](https://vike.dev/render-modes#html-only) mode in this case 🤔

**Reply by magne4000** [reply]
Oh I just understood what you meant 😅, the `vike` package and plugin is not even there, that's definitely an issue!

**Reply by knownasilya** [reply]
Exactly. Yeah I agree an HTML-only might be nice along-side a Vike option.

**Reply by magne4000** [reply, marked as answer]
https://github.com/batijs/bati/pull/161 fixes this issue.
It now correctly adds `vike` as a dependency, adds it to `vite` config, and also gives a few examples to kickstart a project with `onRenderHtml` in HTML-only mode.
